applications/nrf_desktop/doc/battery_charger.rst

```diff
@@ -0,0 +1,53 @@
+.. _battery_charger:
+
+Battery charger module
+######################
+
+The ``battery_charger`` module is responsible for battery charging control.
+
+Module Events
+*************
+
++------------------+---------------------+---------------------+-------------------------+------------------+
+| Source Module    | Input Event         | This Module         | Output Event            | Sink Module      |
++==================+=====================+=====================+=========================+==================+
+|                  |                     | ``battery_charger`` | ``battery_state_event`` | :ref:`led_state` |
++------------------+---------------------+                     +-------------------------+------------------+
+| :ref:`usb_state` | ``usb_state_event`` |                     |                         |                  |
++------------------+---------------------+---------------------+-------------------------+------------------+
+
+Configuration
+*************
+
+The module implemented in ``battery_charger.c`` uses the Zephyr :ref:`zephyr:gpio` driver to control and monitor battery charging. For this reason, you should set :option:`CONFIG_GPIO` option.
+
+Set option ``CONFIG_DESKTOP_BATTERY_CHARGER_DISCRETE`` to enable the module (by default ``CONFIG_DESKTOP_BATTERY_CHARGER_NONE`` is selected - module is disabled).
+
+Following module configuration options are available:
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PIN`` - pin to which CSO (charging status) output from charger is connected
+* CSO pin pull:
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_NONE`` - disabled (default option)
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_UP`` - up
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_DOWN`` - down
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_PIN`` - pin which enables battery charging
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED`` - inverse charging enable signal
```

Comment:
```suggestion
* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED`` - option for enabling the inverse charging.
```
Is `inverse charging` clear to the reader? Would it make sense to introduce it and explain it somewhere in the document?

Reply:
Just signal which enables charging is inversed (charging works still in the same way), I would prefer to leave it as it is.